scripts/pybind/retval.py

```diff
@@ -103,7 +103,7 @@ def cpp_type(self):
     def parse_tuple_format(self):
         return {
             'i': 'i',
-            'u64': 'k',
+            'u64': 'K',
```

Comment:
Is there a particular reason why line 106 would need to be changed but not line 186?

Reply:
From a quick look it seems the other one is not used, but better change it for consistency.